Sources/Build/LLBuildManifestBuilder.swift

```diff
@@ -733,6 +741,22 @@ extension LLBuildManifestBuilder {
             arguments: moduleWrapArgs
         )
     }
+
+    private func addSwiftGetVersionCommand(buildParameters: BuildParameters) -> AbsolutePath {
+        let swiftCompilerPath = buildParameters.toolchain.swiftCompilerPath
+
+        // If we are already tracking this compiler, we can re-use the existing command by just returning the tracking file.
+        if let swiftVersionFilePath = swiftGetVersionFiles[swiftCompilerPath] {
+            return swiftVersionFilePath
+        }
+
+        // Otherwise, come up with a path for the new file and generate a command to populate it.
+        let swiftCompilerPathHash = String(swiftCompilerPath.pathString.hash, radix: 16, uppercase: true)
```

Comment:
is this hash stable?

Reply:
Yah, it's just the hash of `String`.

Reply:
It also doesn't *have* to be stable, though -- we generate it once during build planning and any client receives the file path explicitly.

I just added the hash to avoid collisions between multiple different paths to `swiftc` that may be at play in a single build theoretically.